File: 8_kyu/Correct_the_mistakes_of_the_character_recognition_software.jl
function correct(string::AbstractString)::AbstractString
    function correct(c::Char)::Char
        c == '0' && return 'O'
        c == '1' && return 'I'
        c == '5' && return 'S'
                    return c
    end
    map(correct,string)
end

File: 8_kyu/Correct_the_mistakes_of_the_character_recognition_software.py
def correct(s: str) -> str:
    """
    Corrects mistackes in string.
    Returns string with translated chars 5->S,0->I,1->I.
    """
    return s.translate(CORRECTION_TABLE)

# Table to correct invalid scanned letters.
CORRECTION_TABLE = str.maketrans("501","SOI")
